docs/2.0/getstarted.adoc

```diff
@@ -18,12 +18,12 @@ sudo apt-get install allure-commandline
 ----
 
 === Mac OS X
-For Mas OS, automated installation is available via Homebrew
+For Mas OS, automated installation is available via link:https://brew.sh[Homebrew]
 [source, bash]
 ----
-brew tap qatools/formulas
-brew install allure-commandline
+brew install allure
 ----
+Confirm with `allure --version` in a new shell.
```

Comment:
You can add post install section to check the success of the installation instead of checking for each system
```=== Check the installation
Execute allure --version in console to make sure that allure is now available
> allure --version
 2.0
```